FWCore/Common/interface/SubProcessBlockHelper.h

```diff
@@ -0,0 +1,35 @@
+#ifndef FWCore_Common_SubProcessBlockHelper_h
+#define FWCore_Common_SubProcessBlockHelper_h
+
+/** \class edm::SubProcessBlockHelper
+
+\author W. David Dagenhart, created 4 January, 2021
+
+*/
+
+#include "DataFormats/Provenance/interface/ProvenanceFwd.h"
+#include "FWCore/Common/interface/ProcessBlockHelperBase.h"
+
+#include <vector>
+
+namespace edm {
+
+  class SubProcessBlockHelper : public ProcessBlockHelperBase {
+  public:
+    ProcessBlockHelperBase const* topProcessBlockHelper() const override;
```

Comment:
Is `final` more appropriate?

Reply:
Done. I changed override to final. This is the other class that inherits from ProcessBlockHelperBase.